scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -230,8 +264,8 @@ task LoadData {
   >>>
   runtime {
     docker: docker
-    memory: "4 GB"
-    disks: "local-disk 10 HDD"
+    memory: "10 GB"
+    disks: "local-disk 1000 HDD"
```

Comment:
Why does loading the data need so much disk? Isn't it just importing what's in a separate google bucket?

Reply:
no idea -- I changed it to match Terra, but you're right.  I'm going to change it back :D